boards/get_platform.sh

```diff
@@ -5,7 +5,7 @@ VITIS_PLATFORM=$2
 DOWNLOAD=$3
 PFM_NAME=$(echo $2 | rev | cut -d'/' -f1 | rev | cut -d'.' -f1)
 CURDIR=$(pwd)
-URL="https://www.xilinx.com/bin/public/openDownload?filename=${PFM_NAME}_2019.2.zip"
+URL="https://www.xilinx.com/bin/public/openDownload?filename=${PFM_NAME}_2020.1.zip"
```

Comment:
I think we can get rid of the download method since the old URL does not work any more.